Solving Issue #943 with a Tip note (#1006)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md

3. Next, you should remove the Service Connection Point (SCP) values on your Exchange servers. This ensures that no SCP's are returned, and the client will instead use the DNS method for Autodiscover. An example is shown below:

>    ```Powershell

Reviewer: ```suggestion
   ```PowerShell
```

   2. Run the following command:

>       ```Powershell

Reviewer: ```suggestion
      ```PowerShell
```

   2. Run the following command:

>       ```Powershell

Reviewer: ```suggestion
      ```PowerShell
```

![State before decommissioning Exchange servers](media/c692f0af-6536-4bc9-950d-58a1e486525f.jpg)

>[!TIP]

> > In case that you decide to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud. 

Reviewer: ```suggestion
> If you choose to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud. Each service will authenticate users independently: 
```

> > So each service will authenticate users independenly: 

Reviewer: ```suggestion
```

> > * **O365 identity service** will manage online request 

Reviewer: ```suggestion
> * **Office 365 identity services** will manage online requests
```

> > * **Active directory** will manage the internal authentication

Reviewer: ```suggestion
> * **Azure AD** will manage internal authentication
```

> 

> >  If you don't have any mailbox on-premises, you can safely decommission your exchange server(s).

Reviewer: ```suggestion
>  If you don't have any on-premises mailbox(es), you can safely decommission your exchange server(s).
```

3. Next, you should remove the Service Connection Point (SCP) values on your Exchange servers. This ensures that no SCP's are returned, and the client will instead use the DNS method for Autodiscover. An example is shown below:

>    ```Powershell

Reviewer: ```suggestion
   ```PowerShell
```

4. To prevent the hybrid configuration objects from being recreated in the future, you should remove the hybrid configuration object from Active Directory. To do this, open the Exchange Management Shell and run the following:

>    ```Powershell

Reviewer: ```suggestion
   ```PowerShell
```

   2. Run the following command:

>       ```Powershell

Reviewer: ```suggestion
      ```PowerShell
```

   2. Run the following command:

>       ```Powershell

Reviewer: ```suggestion
      ```PowerShell
```
